src/lamoda/routers.py

```diff
@@ -0,0 +1,33 @@
+from decimal import Decimal
+from typing import Annotated
+
+from bson import Decimal128
+from database import Collection, get_lamoda_collection
+from fastapi import APIRouter, Depends
+
+from .service import parse_object
+
+lamoda_router = APIRouter(prefix='/lamoda')
+
+LamodaCollection = Annotated[Collection, Depends(get_lamoda_collection)]
+
+
+@lamoda_router.post("/product")
+def parse_product(url: str, collection: LamodaCollection):
+    """View for parsing product by its url"""
+
+    product = parse_object(url)
+    Decimal128(str(product.price))
+    product_as_dict = product.dict()
+    for key, value in product_as_dict.items():
+        if isinstance(value, Decimal):
+            product_as_dict[key] = str(value)
+    product.price = str(product.price)
+    created_id = collection.insert_one(product_as_dict)  # it is temporary, made to check the ability to save into db
+    created = collection.find_one({"_id": created_id.inserted_id})  # soon will make other alg to save into db
+    return product.dict()
```

Comment:
any reason why don't you use schemas? see a great example of the project -> https://github.com/zhanymkanov/fastapi_production_template/blob/main/src/auth/router.py#L50-L66